src/PowerManifold.jl

```diff
@@ -525,6 +528,45 @@ function exp!(M::PowerManifoldNestedReplacing, q, p, X)
     return q
 end
 
+@doc raw"""
+    fill(p, M::AbstractPowerManifold)
+
+Create a point on the [`AbstractPowerManifold`](@ref) `M`, where every entry is set to the
+point `p`.
+
+!!! note
+    while usually the manifold is a first argument in all functions in `ManifoldsBase.jl`,
+    we follow the signature of `fill`, where the power manifold serves are the size information.
+"""
+function fill(p, M::AbstractPowerManifold)
+    P = allocate_result(M, rand) # rand finds the right way to allocate our point usually
+    return fill!(P, p, M)
+end
+
+@doc raw"""
+    fill!(P, p, M::AbstractPowerManifold)
+
+Fill a point `P` on the [`AbstractPowerManifold`](@ref) `M`, setting every entry to `p`.
+
+!!! note
+    while usually the manifold is the first argument in all functions in `ManifoldsBase.jl`,
+    we follow the signature of `fill!`, where the power manifold serves are the size information.
+"""
+function fill!(P, p, M::PowerManifoldNestedReplacing)
+    for i in get_iterator(M)
+        P[i...] = p # can we do something that this is closer to fill/fill! and set the ref to p?
```

Comment:
What do you mean by this?

Reply:
fill/fill! are documented to have the same reference in all elements, I think this one actually copies?

Reply:
No, this doesn't make any copies :slightly_smiling_face: . It's exactly the same as `fill`/`fill!`.

Reply:
Ok, then I'll remove that comment.